css/index.css

```diff
@@ -1,3 +1,349 @@
+  display: inline-block;
+  width: 691px;
+  vertical-align: top;
+  margin-top: 35px;
+}
+
+header nav a {
+  display: inline-block;
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: inline-block;
+  margin-top: 25px;
+}
+
+/* cta */
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* media queries */
+
+/* tablet */
+
+@media (max-width: 800px) {
+  .container {
+    width: 800px;
+   }
+  .cta img {
+    display: none;
+  }
+  .main-content .middle-img {
+    width: 780px;
+    padding: 0 0 0 20px;
+    
+  }
+  header {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+  header nav {
+    display: flex;
+    order: 2;
+    justify-content: space-evenly;
+    width: 800px;
+  }
+
+  header img.logo {
+    display: flex;
+    justify-content: center;
+    margin: 0 auto;
+    margin-top: 20px;
+    order: 1;
+  }
+  .cta .cta-text {
+    display: flex;
+    justify-items: center;
+    flex-direction: column;
+    margin: 50px;
+  }
+  .cta .cta-text button {
+    margin: 0 auto;
+  }
+  .main-content .top-content {
+    display: flex;
+  }
+  .main-content .bottom-content {
+    display: flex;
+  }
+}
+
+@media (max-width: 500px) {
+  .container {
+    width: 500px;
+  }
+  header nav {
+    width: 500px;
+    flex-direction: column;
+    border-top: 1px solid black;
+    margin-top: 15px;
+  }
+  header nav a {
+    width: 100%;
+    justify-content: center;
+    display: flex;
```

Comment:
There is no need to repeat display: flex in media queries if that rule is already defined at the top of the file. This applies to every rule in CSS. If You have defined something at the top, there is no need to repeat that in media queries. You only do it if You want particular thing to actually change on that breakpoint.

For example if You had display: flex, You only need to use `display` again if You want to change flex to something else. Adhering to that rule makes it easier to follow the DRY (Don't Repeat Yourself) principle.